pkgs/tools/misc/nginx-config-formatter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  version = "2016-06-16";
+  name = "nginx-config-formatter-${version}";
```

Comment:
Yeah, but later when, e.g., version 1.0 is released I believe `nix-env -u` will break anyway since `1.0 < 2016-06-16` and the user would have to manually force a package update.